### PR TITLE
ERROR_INFORMATION variable added in zeebe variables

### PR DESCRIPTION
**Pull request by danishjamal104**

### CODE REVIEW

File: src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java [hunk 1]
```diff
@@ -65,11 +65,18 @@ public void setupWorkers() {
 
                         producerTemplate.send("direct:transfer-validation-base", ex);
 
-                        boolean isPartyLookUpFailed = ex.getProperty(PARTY_LOOKUP_FAILED, boolean.class);
+                        Boolean isPartyLookUpFailed = ex.getProperty(PARTY_LOOKUP_FAILED, Boolean.class);
                         variables.put(PARTY_LOOKUP_FAILED, isPartyLookUpFailed);
+                        if(isPartyLookUpFailed != null && isPartyLookUpFailed) {
+                            variables.put(ERROR_INFORMATION, ex.getIn().getBody(String.class));
+                            // TODO: add [ERROR_CODE] and [ERROR_DESCRIPTION] variable in zeebe after parsing [ERROR_INFORMATION]
```
Comment (danishjamal104) [inline src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java]: @avikganguly01 once gilbert will share the payload ill parse the `ERROR_INFORMATION` and update the `ERROR_CODE` and `ERROR_DESCRIPTION` variable.

File: src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java [hunk 2]
```diff
@@ -99,11 +106,18 @@ public void setupWorkers() {
 
                         producerTemplate.send("direct:transfer-settlement", ex);
 
-                        boolean isSettlementFailed = ex.getProperty(TRANSFER_SETTLEMENT_FAILED, boolean.class);
+                        Boolean isSettlementFailed = ex.getProperty(TRANSFER_SETTLEMENT_FAILED, Boolean.class);
                         variables.put(TRANSFER_SETTLEMENT_FAILED, isSettlementFailed);
+                        if(isSettlementFailed != null && isSettlementFailed) {
+                            variables.put(ERROR_INFORMATION, ex.getIn().getBody(String.class));
+                            // TODO: add [ERROR_CODE] and [ERROR_DESCRIPTION] variable in zeebe after parsing [ERROR_INFORMATION]
```
Comment (danishjamal104) [inline src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java]: same here

**Comment by danishjamal104** [comment]
@avikganguly01 and @SubhamPramanik please review and merge. Thanks :)

File: src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java (hunk 1 above)
Comment (avikganguly01) [reply src/main/java/org/mifos/connector/ams/pesacore/zeebe/ZeebeWorkers.java]: I guess Gilbert mentioned it looks like this.

 { "400": {
                 "description": "Bad Request, incorrect data or missing required data"
             },
             "401": {
                 "description": "check API authentication"
             },
             "404": {
                 "description": "Client Not Found"
             }
}

**Review by avikganguly01 — CHANGES_REQUESTED** [review]
Attached Gilbert's 400 response json stricture.

**Comment by danishjamal104** [comment]
> Attached Gilbert's 400 response json stricture.

yup

**Review by avikganguly01 — APPROVED** [review]

**Pull request merged**